### PR TITLE
Update performance test agent label

### PR DESCRIPTION
**Pull request by kotwanikunal**
Signed-off-by: Kunal Kotwani <kkotwani@amazon.com>

### Description
- Update the agent label for performance tests
 
### Issues Resolved
- https://github.com/opensearch-project/opensearch-build/issues/129
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: jenkins/opensearch/perf-test.jenkinsfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ pipeline {
         timeout(time: 10, unit: 'HOURS')
     }
     environment {
-        AGENT_LABEL = 'Jenkins-Agent-al2-x64-c54xlarge-Docker-Host'
+        AGENT_LABEL = 'Jenkins-Agent-al2-x64-m5xlarge-Perf-Test'
```
Comment (gaiksaya) [inline jenkins/opensearch/perf-test.jenkinsfile:9]: Can you confirm that this agent configuration exists in the current  jenkins set up before we merge?
Thanks!

**Review by gaiksaya — APPROVED** [review]

File: jenkins/opensearch/perf-test.jenkinsfile (hunk 1 above)
Comment (kotwanikunal) [reply jenkins/opensearch/perf-test.jenkinsfile:9]: I did check the server for the agent and it does exist. I tried to run a sample test which failed though.
Error - 
```
+ docker pull opensearchstaging/ci-runner:ci-runner-centos7-v1
/var/jenkins/workspace/perf-tests-dev/perf-test-v3@tmp/durable-effa1e97/script.sh: line 1: docker: command not found
```
cc: @peterzhuamazon

Comment (gaiksaya) [reply jenkins/opensearch/perf-test.jenkinsfile:9]: Yeah I checked and looks like docker installation commands are missing. Is someone is taking care of it? If not I can add it.

Comment (peterzhuamazon) [reply jenkins/opensearch/perf-test.jenkinsfile:9]: @kotwanikunal like I mentioned before you dont use docker on that particular host.
You directly use that agent therefore no docker needed.
Thanks.

Comment (kotwanikunal) [reply jenkins/opensearch/perf-test.jenkinsfile]: Synced up offline. Resorted to using docker for this use case.

**Comment by kotwanikunal** [comment]
@opensearch-project/engineering-effectiveness This is tested and ready to be merged in.

**Pull request merged**